pootle/core/views.py

```diff
@@ -74,9 +74,11 @@ def set_permissions(f):
     @functools.wraps(f)
     def method_wrapper(self, request, *args, **kwargs):
         User = get_user_model()
-        request.profile = User.get(self.request.user)
```

Comment:
@julen, as I understand we made extra query here. Now we make this query for anonymous only. This is optimisation, is this?